README.md

```diff
@@ -34,14 +34,18 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+To run this app you will also need to sign up for a trello account (https://trello.com/signup). By doing so you can then generate and API key and a token. You will need to add these as values in your `.env` file.
```

Comment:
You should update the `.env.template` file with the keys and placeholder values that you use